lib/batchVersions.js

```diff
@@ -70,8 +70,7 @@ export async function create({id, tokenizerId, options}) {
     }
   };
   try {
-    const result = await collection.insertOne(record, database.writeOptions);
-    record = result.ops[0];
```

Comment:
`record` here was not being used and therefore result was unused.

Reply:
I think the problem here is that we weren't returning `record`. So we should change it to do that instead.